app/src/main/java/com/github/ybecker/epforuml/database/Database.kt

```diff
@@ -135,15 +135,15 @@ abstract class Database {
      * @param userId the id of the user that endorsed
      * @param questionId the id of the question that is endorsed
      */
-    abstract fun addQuestionEndorsement(userId:String, questionId: String)
+    abstract fun addQuestionFollower(userId:String, questionId: String)
```

Comment:
I like the idea of the Like button, however I feel like endorsement for teachers is still good as it makes a good reference about the question